### PR TITLE
Improvements to Getting Started guide

### PR DESCRIPTION
**Pull request by mmattar**
- Changed the ordered list to use "1."
- Trimmed down text
- Removed references to Agent APIs

### Proposed change(s)

Describe the changes made in this PR.

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [X] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [X] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: docs/Getting-Started.md [hunk 1]
```diff
@@ -145,88 +94,45 @@ We include pre-trained models for our agents (`.nn` files) and we use the
...
    ![Platform Prefab](images/platform_prefab.png)
 
-3. In the **Project** window, drag the **3DBall** Model located in
+1. In the **Project** window, drag the **3DBall** Model located in
    `Assets/ML-Agents/Examples/3DBall/TFModels` into the `Model` property under `Behavior Parameters (Script)` component in the Agent GameObject **Inspector** window.
 
    ![3dball learning brain](images/3dball_learning_brain.png)
 
-4. You should notice that each `Agent` under each `3DBall` in the **Hierarchy** windows now contains **3DBall** as `Model` on the `Behavior Parameters`. __Note__ : You can modify multiple game objects in a scene by selecting them all at
+1. You should notice that each `Agent` under each `3DBall` in the **Hierarchy** windows now contains **3DBall** as `Model` on the `Behavior Parameters`. __Note__ : You can modify multiple game objects in a scene by selecting them all at
```
Comment (vincentpierre) [inline docs/Getting-Started.md:109]: This is a long line

File: docs/Getting-Started.md [hunk 2]
```diff
@@ -77,66 +62,30 @@ the 3D Balance Ball environment, the Agent components are placed on the twelve
...
+This means that the feature vector containing the Agent's observations contains
+eight elements: the `x` and `z` components of the agent cube's rotation and the
+`x`, `y`, and `z` components of the ball's relative position and velocity.
 
 #### Behavior Parameters : Vector Action Space
 
 An Agent is given instructions in the form of a float array of *actions*.
-ML-Agents toolkit classifies actions into two types: the **Continuous** vector
-action space is a vector of numbers that can vary continuously. What each
-element of the vector means is defined by the Agent logic (the training
```
Comment (vincentpierre) [inline docs/Getting-Started.md:131]: Is this paragraph explained somewhere else?

File: docs/Getting-Started.md [hunk 3]
```diff
@@ -260,24 +166,26 @@ INFO:mlagents_envs:Hyperparameters for the PPO Trainer of brain 3DBallLearning:
...
-INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 3000. Mean Reward: 1.804. Std of Reward: 1.056. Training.
-INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 4000. Mean Reward: 2.151. Std of Reward: 1.432. Training.
-INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 5000. Mean Reward: 3.175. Std of Reward: 2.250. Training.
-INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 6000. Mean Reward: 4.898. Std of Reward: 4.019. Training.
-INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 7000. Mean Reward: 6.716. Std of Reward: 5.125. Training.
-INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 8000. Mean Reward: 12.124. Std of Reward: 11.929. Training.
-INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 9000. Mean Reward: 18.151. Std of Reward: 16.871. Training.
-INFO:mlagents.trainers: first-run-0: 3DBallLearning: Step: 10000. Mean Reward: 27.284. Std of Reward: 28.667. Training.
+        model_path: ./models/first-3dball-run-0/3DBallLearning
+INFO:mlagents.trainers: first-3dball-run-0: 3DBallLearning: Step: 1000. Mean Reward: 1.242. Std of Reward: 0.746. Training.
```
Comment (vincentpierre) [inline docs/Getting-Started.md:175]: There are dashes in `first-3dball-run` but not in the run-id given above.

File: docs/Getting-Started.md (hunk 1 above)
Comment (mmattar) [reply docs/Getting-Started.md:109]: Done.

File: docs/Getting-Started.md (hunk 2 above)
Comment (mmattar) [reply docs/Getting-Started.md:131]: In the Designing Agents file.

**Pull request merged**